experimental/GModule/GaloisCohomology.jl

```diff
@@ -1514,6 +1511,30 @@ an infinite direct sum of the local Brauer groups.
 
 The second return value is a map translating between the local data
 and explicit 2-cochains.
+
+```jldoctest
+julia> G = SL(2,5)
+SL(2,5)
+
+julia> T = character_table(G);
+
+julia> R = gmodule(T[9])
+G-module for G acting on vector space of dimension 6 over abelian closure of Q
+
+julia> S = gmodule(CyclotomicField, R)
+G-module for G acting on vector space of dimension 6 over cyclotomic field of order 5
+
+julia> B, mB = relative_brauer_group(base_ring(S), character_field(S));
+
+julia> B
+Relative Brauer group for cyclotomic field of order 5 over number field of degree 1 over QQ
+
+julia> b = B(S)
+Element of relative Brauer group of number field of degree 1 over QQ
+  <2, 2> -> 1//2 + Z
+  Complex embedding of number field -> 1//2 + Z
+  <5, 5> -> 0 + Z
```

Comment:
Seems the order in which these three lines are printed is not stable, hence some of the doctest CI jobs fail :/

Reply:
I don't see a good way to sort the keys *in general*, they are ideals in a number field, or field embeddings. But we could make sure that the ideals come first. And perhaps the ideals could be sorted by minimum or norm. That won't give a unique sorting in all cases, but it is good enough for this example and it probably is nice for the user overall. Will do that for now.


@fieker I also wonder if we should print something nicer instead of "Complex embedding of number field"?